### ISSUE TITLE
FvwmButtons subpanels not popped out on primary monitor when desk is > 0

### ISSUE THREAD

**Issue by NsCDE** [fvwmorg/fvwm3#224]
Hi,

I have tried this one with FVWM2, and it is not some long standing bug.

There is main FvwmButtons panel "front panel" ([configuration](https://github.com/NsCDE/NsCDE/blob/fvwm3_support/NsCDE/config/NsCDE-FrontPanel.conf)) and it has launchers for transient [subpanels](https://github.com/NsCDE/NsCDE/blob/fvwm3_support/NsCDE/config/NsCDE-Subpanels.conf). This works ok with one monitor, but on two-monitor configuration, subpanels are not shown on the primary monitor if desk is changed from 0 to 1, 2 or 3. After playing around, even on desk 0 subpanels are popping out someware outside of the screen.

If main "FrontPanel" is moved to secondary screen, things are working on all 4 configured desks.

All described observations are recorded in this video for better understanding:
https://youtu.be/-BHCVN4U40s

**Comment by ThomasAdam** [comment]
Hi @NsCDE 

Thanks. I presume this happens regardless of how `DesktopConfiguration` is set?

BTW, why don't you join us on IRC? #fvwm on freenode

-- Thomas

**Comment by NsCDE** [comment]
Hi @ThomasAdam 

1. Yes, just tried DesktopConfiguration per-monitor and this does not seems to be relevant in this case.

2. Thanks for the invitation. I was USENet user long ago, IRC somehow passed beside me. I can try to find some IRC client. :)

**Comment by ThomasAdam** [comment]
Hi @NsCDE,

Please try this patch on top of `master`:

```
diff --git a/modules/FvwmButtons/FvwmButtons.c b/modules/FvwmButtons/FvwmButtons.c
index 7c06a406a..feae50fbb 100644
--- a/modules/FvwmButtons/FvwmButtons.c
+++ b/modules/FvwmButtons/FvwmButtons.c
@@ -1989,7 +1989,7 @@ static void HandlePanelPress(button_info *b)
     XSetWMHints(Dpy, b->PanelWin, &wmhints);
 
     /* map the window in the void */
-    XMoveWindow(Dpy, b->PanelWin, 32767, 32767);
+    /*XMoveWindow(Dpy, b->PanelWin, 32767, 32767);*/
     XMapWindow(Dpy, b->PanelWin);
     XSync(Dpy, 0);
```

**Comment by NsCDE** [comment]
Hi @ThomasAdam 

Yes, this makes a change and fixes this behaviour half way, but introduces really interesting behaviour, let's try without mp4 this time:

* Subpanels are opened normally on both monitors and all existing desks
* If main front panel is moved to secondary monitor, subpanels appear on the screen every second time
* If main front panel is moved back, behaviour from step above remains - every second time
* If desk is changed, this "every second time" behaviour stops and everything is normal until front panel is moved to another monitor
* this new behaviour is the same regardless of the left/right monitor: when main front panel is moved to another screen, subpanels are opening every second time. Changing active desk stops this, and subpanels are opened every time on every desk, until main front panel is again relocated to another monitor.

**Comment by ThomasAdam** [comment]
Hi @NsCDE,

Can you try the ` ta/panel-map` branch, and let me know if that fixes this issue?  I believe it to...

**Comment by NsCDE** [comment]
Hi Thomas, sorry unplanned busy day on work yesterday.

Compiled, tried ... success. This fixes #224 according to my tests now.
